### PR TITLE
TiKV txn: retry retryable error on update

### PR DESCRIPTION
**Pull request by yiwu-arbug**
For TiKV txn API benchmarks, retry retriable errors on txn.Commit(). See #94 for more details.

### CODE REVIEW

File: db/tikv/txn.go [hunk 1]
```diff
@@ -153,43 +154,43 @@ func (db *txnDB) Scan(ctx context.Context, table string, startKey string, count
 }
 
 func (db *txnDB) Update(ctx context.Context, table string, key string, values map[string][]byte) error {
-	rowKey := db.getRowKey(table, key)
+	// Suppress RunInNewTxn warning log.
+	log.SetLevel(log.ErrorLevel)
```
Comment (yiwu-arbug) [inline db/tikv/txn.go]: This is to suppress error printed from the tidb/kv package, but I think it should move to a better place and outside of this PR.

File: db/tikv/txn.go [hunk 2]
```diff
@@ -153,43 +154,43 @@ func (db *txnDB) Scan(ctx context.Context, table string, startKey string, count
 }
 
 func (db *txnDB) Update(ctx context.Context, table string, key string, values map[string][]byte) error {
-	rowKey := db.getRowKey(table, key)
+	// Suppress RunInNewTxn warning log.
```
Comment (yiwu-arbug) [inline db/tikv/txn.go]: Other than the Update operation, I'm not sure if other operations needs retry logic. I would like to get some feedback on whether we need to retry at all.

File: db/tikv/txn.go (hunk 1 above)
Comment (siddontang) [reply db/tikv/txn.go]: yes, I think we don't need logging. so you don't need to import log module.

Comment (siddontang) [reply db/tikv/txn.go]: or if you want to suppress these logs, I think we should set level at first or in `init` function, not in update function.

**Comment by siddontang** [comment]
now we have used client-go, I will close this PR and may use another way later.

**Pull request closed without merging**